kinto/core/authentication.py

```diff
@@ -34,6 +34,7 @@ def unauthenticated_userid(self, request):
                 return
 
             hmac_secret = settings["userid_hmac_secret"]
+            # Unsure how to use fstring with *args
```

Comment:
I would say that `f"{credentials[0]}:{credentials[1]"` is fine. Or leave the format() call